core/Tracker/Visit.php

```diff
@@ -397,6 +397,11 @@ private static function toCanonicalHost($host)
      */
     protected function updateExistingVisit($valuesToUpdate)
     {
+        if (empty($valuesToUpdate)) {
```

Comment:
This is bit off topic but fixes an issue if a plugin unsets all values to be updated eg in a requestprocessor.